lib/main.dart

```diff
@@ -12,7 +12,7 @@ class MainApp extends StatelessWidget {
     return const MaterialApp(
       home: Scaffold(
         body: Center(
-          child: Text('Hello World!'),
+          child: Text('Hello Emmanuel!'),
```

Comment:
```suggestion
          child: Text('Hello Emmanuel!, style:TextStyle()'),
```

Add styles to your text